src/providers/tfc/runProvider.ts

```diff
@@ -44,6 +43,13 @@ export class RunTreeDataProvider implements vscode.TreeDataProvider<TFCRunTreeIt
         this.reporter.sendTelemetryEvent('tfc-runs-viewInBrowser');
         vscode.env.openExternal(run.websiteUri);
       }),
+      vscode.commands.registerCommand('terraform.cloud.run.viewPlan', async (run: PlanTreeItem) => {
+        if (!run.id) {
+          await vscode.window.showErrorMessage(`No plan found for ${run.id}`);
+          return;
+        }
+        this.planDataProvider.refresh(run.id);
```

Comment:
```suggestion
        await vscode.commands.executeCommand('terraform.cloud.run.plan.focus');
        this.planDataProvider.refresh(run.id);
```

If the user hasn't already opened this panel, then clicking this button results in no visual change. The plan is queried and data successfully retrieved, but no data is displayed because the panel isn't visible. There isn't a direct method on a TreeView to change visibility, but you can use an `executeCommand` method to set focus to our panel. This works because all views have a set of commands automatically registered for them by VS Code to control things like visiblity.

Reply:
Thanks, good catch and great suggestion! 😅 

Are these "magic" command ID "suffixes" (like `.focus`) documented anywhere, so I know where to look next time?

Reply:
They are technically documented. If you open an editor with the terraform extension activated and run Open Default Keyboard Shortcuts, you get a list of all keyboard shortcuts.

<img width="720" alt="image" src="https://github.com/hashicorp/vscode-terraform/assets/272569/0c1dd625-a0e8-437b-9447-2ba837a8dffa">

At the bottom is a commented out list of all commands that are available in this session to bind. Search for our extension identifier, and you will see the ones we contribute along with the ones generated for our views:

<img width="756" alt="image" src="https://github.com/hashicorp/vscode-terraform/assets/272569/06fae016-b64f-491b-82fe-7d9d7538fd34">